packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css

```diff
@@ -117,6 +125,11 @@
   z-index: 1;
 }
 
+.list-item:hover .icon {
+  /* 1.16 is roughly the ratio of icon-size-hover to --icon-size */
+  transform: scale(1.2);
+}
+
```

Comment:
@ddouglasz sorry just noticed one more tiny thing, I think this block is now duplicated to the one in line 102?